passes/cmds/scratchpad.cc

```diff
@@ -30,14 +31,18 @@ struct ScratchpadPass : public Pass {
 	{
 		//   |---v---|---v---|---v---|---v---|---v---|---v---|---v---|---v---|---v---|---v---|
 		log("\n");
-		log("    scratchpad [ -get id | -set id val | -unset id | -copy id1 id2 ]\n");
+		log("    scratchpad [options]\n");
 		log("\n");
 		log("This pass allows to read and modify values from the scratchpad of the current\n");
 		log("design. Options:\n");
 		log("    -get <identifier>\n");
+		log("        print the value saved in the scratchpad under the given identifier\n");
 		log("    -set <identifier> <value>\n");
```

Comment:
Looks like you're still missing newlines in between the options, and a full stop at the end of the description

Reply:
Oh, right. Starting with a lower case letter but ending with a period is kind of a counterintuitive style choice...